01-atoms/links/Link.component.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withModifiers from '../../_utils/withModifiers';
+
+const Link = ({children, modifiers, linkURL, linkTarget, linkTitleText }) => {
+  return (
+    <a
+      href={linkURL}
+      target={linkTarget}
+      title={linkTitleText}
```

Comment:
Everything is done correctly. This may just look a little redundant. We're already in the context of a link component. I'm not sure there is a reason to prefix the name of every attribute with link*. It has just as much meaning and is more concise without it. Just a suggestion. Take or leave it.